src/ucs/debug/log.c

```diff
@@ -478,6 +478,31 @@ const char *ucs_log_bitmap_to_str(unsigned n, uint8_t *bitmap, size_t length)
     return buf;
 }
 
+void __ucs_log_wqe(const char *file, unsigned line,
+                   const char *function, ucs_log_level_t level,
+                   ucs_log_component_config_t *comp_conf,
+                   const void* buff, size_t len)
+{
+    const uint32_t *p = (const uint32_t *)buff;
+    size_t buflen     = 41 * ucs_div_round_up(len, 16);
```

Comment:
Maybe add some basic comment/static calculation for that magic 41 value?

Reply:
```
xx: xxxxxxxx xxxxxxxx xxxxxxxx xxxxxxxx\n
|-------------------41------------------|
```

Reply:
Something like that..